utils/python/CIME/SystemTests/erp.py

```diff
@@ -11,134 +11,82 @@
-            self.run_indv(suffix=suffix)
+        SystemTestsCompareTwo.__init__(self, case,
+                                       separate_builds = True,
+                                       run_two_suffix = "BFBRestTest",
+                                       run_one_st_archive = True,
+                                       run_one_description = "Baseline run from day 0 to STOP_N",
+                                       run_two_description = ("Test run starting from run one's " +
+                                                              "state at STOP_N // 2 + 1 to STOP_N.\n" +
+                                                              "Run 2 is done with half of the " +
+                                                              "number of tasks and threads as run 1"))
+
+    def _case_one_setup(self):
+        stop_n = self._case.get_value("STOP_N")
+        rest_n = _compute_rest_n(stop_n)
+        expect(stop_n > rest_n , "STOP_N value too small for test")
+        self._case.set_value("REST_N", rest_n)
+        self._case.set_value("REST_OPTION", self._case.get_value("STOP_OPTION"))
+
+    def _case_two_setup(self):
+        """ Case two uses half the number of threads and tasks as the defaults and case one. """
+        self._case.set_value("REST_OPTION", "never")
+
+        stop_n_1 = self._case1.get_value("STOP_N")
+        stop_n = stop_n_1 - _compute_rest_n(stop_n_1)
+        expect(stop_n > 0, "STOP_N value too small for test")
+        self._case.set_value("STOP_N", stop_n)
+
+        self._case.set_value("CONTINUE_RUN", True)
+
+        for comp in self._case.get_values("COMP_CLASSES"):
+            if comp == "DRV":
+                comp = "CPL"
+            ntasks = self._case.get_value("NTASKS_%s"%comp)
+            if ntasks > 1:
+                rootpe = self._case.get_value("ROOTPE_%s"%comp)
+                self._case.set_value("NTASKS_%s"%comp, ntasks // 2)
+                self._case.set_value("ROOTPE_%s"%comp, rootpe // 2)
+            nthreads = self._case.get_value("NTHRDS_%s"%comp)
+            if nthreads > 1:
+                self._case.set_value("BUILD_THREADED", True)
+                self._case.set_value("NTHRDS_%s"%comp, nthreads // 2)
+        case_setup(self._case, test_mode=True, reset=True)
+
+    def _inter_run_hook(self):
+        st_archive_dir_one = self._case1.get_value("DOUT_S_ROOT")
+        restdir_name = "rest"
+        restdir = os.path.join(st_archive_dir_one, restdir_name)
+        rundir = self._case2.get_value("RUNDIR")
+        path_1 = None
+        for root, _, files in os.walk(restdir):
+            for f in files:
+                if path_1 == None:
+                    path_1 = root
+                fpath_in = os.path.join(root, f)
+                fpath_out = os.path.join(rundir, f)
+                os.symlink(fpath_in, fpath_out)
+
+        ref_case = self._case1.get_value("CASE") + ".ref1"
+        self._case2.set_value("REF_CASE", ref_case)
```

Comment:
This should be RUN_REFCASE, not REF_CASE.

In addition, you need to set RUN_TYPE to branch.

I'd prefer if the setting of RUN_REFCASE and RUN_TYPE were done in _case_two_setup, because I prefer if as much as possible is done there rather than in this new _inter_run_hook. It does seem like you need to wait until this hook to set RUN_REFDATE, though. So I don't feel strongly about this.